README.md

```diff
@@ -207,22 +207,22 @@ By default, all possible comparisons are performed. Use `--deg` to change this.
 
 #### Read files (required)
 
-Specify your read files in `FASTQ` format with `--reads input.csv`. The file `input.csv` has to look like this for single-end reads:
+Specify your read files in `FASTQ` format with `--reads input.csv`. The file `input.csv` has to look like this for single-end reads (just leave R2 empty):
 
 ```csv
-Sample,R,Condition,Source
-mock_rep1,/path/to/reads/mock1.fastq.gz,mock,
-mock_rep2,/path/to/reads/mock2.fastq.gz,mock,
-mock_rep3,/path/to/reads/mock3.fastq.gz,mock,
-treated_rep1,/path/to/reads/treat1.fastq.gz,treated,
-treated_rep2,/path/to/reads/treat2.fastq.gz,treated,
-treated_rep3,/path/to/reads/treat3.fastq.gz,treated,
+Sample,R1,R2,Condition,Source,Strandedness
+mock_rep1,/path/to/reads/mock1.fastq.gz,,mock,
+mock_rep2,/path/to/reads/mock2.fastq.gz,,mock,
+mock_rep3,/path/to/reads/mock3.fastq.gz,,mock,
+treated_rep1,/path/to/reads/treat1.fastq.gz,,treated,
+treated_rep2,/path/to/reads/treat2.fastq.gz,,treated,
+treated_rep3,/path/to/reads/treat3.fastq.gz,,treated,
 ```
 
 and for paired-end reads, like this:
 
 ```csv
-Sample,R1,R2,Condition,Source
+Sample,R1,R2,Condition,Source,Strandedness
```

Comment:
Here the last comma is missing, or? For the (optional) Strandedness column:

```
mock_rep1,/path/to/reads/mock1_1.fastq,/path/to/reads/mock1_2.fastq,mock,A,
...
```

Reply:
Since its the last column this will not break anything because the strandedness will then just be detected as the default value, but I can add the commas for completeness.